src/data/spotifyDAO.ts

```diff
@@ -20,8 +20,11 @@ export class SpotifyAPI {
         // Retrieve an access token.
         this.api.clientCredentialsGrant().then(
             (data) => {
-                console.log('The access token expires in ' + data.body.expires_in);
-                console.log('The access token is ' + data.body.access_token);
+                console.log('Spotify access token expires in ' + data.body.expires_in);
+                console.log('Spotify access token is ' + data.body.access_token);
+                setTimeout(() => {
+                    console.log('Spotify access token expired!');
+                }, data.body.expires_in * 1000);
```

Comment:
Was gonna ask if you had an issue for this but you do :)

Reply:
Yeah haha, didn't wanna fix it here yet.